modules/vaos/.rubocop.yml

```diff
@@ -0,0 +1,14 @@
+inherit_from:
+  - ../../.rubocop.yml
+
+AllCops:
+  Exclude:
+    - 'bin/*'
+    - bin/rails
+    - db/migrate/*.rb
+    - 'spec/dummy/**/*'
+    - spec/dummy/db/schema.rb
+
+Metrics/BlockLength:
+  Exclude:
+    - spec/**/*.rb
```

Comment:
This is already in the included `rubocop.yml` -- if that doesn't work when you're in the module, maybe we need to hoist `modules/*/spec/**/*.rb` up to that file to avoid including hit in each?